Marlin/src/lcd/ultralcd.cpp

```diff
@@ -521,7 +523,7 @@ uint16_t max_display_update_time = 0;
           if (currentScreen == lcd_status_screen)
             doubleclick_expire_ms = millis() + DOUBLECLICK_MAX_INTERVAL;
         }
-        else if (screen == lcd_status_screen && currentScreen == lcd_main_menu && PENDING(millis(), doubleclick_expire_ms) && (planner.movesplanned() || IS_SD_PRINTING))
+        else if (screen == lcd_status_screen && currentScreen == lcd_main_menu && PENDING(millis(), doubleclick_expire_ms)/* && (planner.movesplanned() || IS_SD_PRINTING)*/)
```

Comment:
How does babystepping behave when there's no active printing?
Did we already add code to activate steppers when babysteps are received? (I think so…)

Reply:
> Did we already add code to activate steppers when babysteps are received? (I think so…)

If that was not removed ... . At least in the initial introduction of babystepping that was included.

Reply:
I have only used that for Z-BabyStepping.   But it works just fine.

Reply:
I can confirm that it works just fine with those additional conditionals removed.